uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-calendar-legend.component.js

```diff
@@ -18,7 +18,15 @@ define([
     $log.debug('Component: leave-calendar-legend');
 
     var vm = this;
+
     vm.legendCollapsed = true;
+    vm.other_badges = [
+      { label: 'AM', hint: 'AM Only' },
```

Comment:
The names are bit confusion in my opinion, how about the following?
`{ iconLabel: '', description: 'Requested', cssClassSuffix: 'requested' }`

Reply:
I think `iconLabel` is more confusing, this is not an icon, it is a badge I would say, so if we keep `otherBadge.label` - should be ok. Agreed with the rest.